javaslang/src/test/java/javaslang/collection/AbstractTraversableTest.java

```diff
@@ -489,10 +489,16 @@ public void shouldDropWhileAllIfPredicateIsTrue() {
     }
 
     @Test
-    public void shouldDropWhileCorrect() {
+    public void shouldDropWhileAccordingToPredicate() {
         assertThat(of(1, 2, 3).dropWhile(i -> i < 2)).isEqualTo(of(2, 3));
     }
 
+    // fixes #1641
```

Comment:
No, I will remove it. What counts is the current state of the application, it reflects the truth. The less organizational overhead, the better. Commits to a file are browsable. Also the full-text search function for issues and PRs is great. Thank you for asking!